docs/jotai/guides/nextjs.mdx

```diff
@@ -16,16 +18,3 @@ const postData = atom((get) => {
 })
 ```
 
-## Hydration is possible with care
-
-Check the following examples.
-
-## Examples
-
-### Clock
-
-<Codesandbox id="5ylrj" />
-
-### HN Posts
-
-<Codesandbox id="pm0l8" />
```

Comment:
These examples are good to show. Would you fork and create useHydrateAtoms versions? Or let me know if you want me to work on them.

Reply:
We should probably fork them 🙂